src/metabase/lib/expression.cljc

```diff
@@ -441,7 +443,7 @@
    (cyclic-definition node->children start []))
   ([node->children node path]
    (if (some #{node} path)
-     (conj path node)
+     (drop-while (complement #{node}) (conj path node))
```

Comment:
This is just reducing the error report to the important part, it's fixing an oversight.